app/helpers/content_helper.rb

```diff
@@ -93,6 +93,6 @@ def opt_in_out(type)
 
   # @yield [String]
   def password_complexity
-    t(:password_complexity, length: User.password_length.first)
+    t('password_complexity.html', length: User.password_length.first)
```

Comment:
I think Content will find it easier reusing keys. It is possible to avoid HTML in the CMS resource which would be preferable.